foundry/smb3parse/util/code_edit_byte.py

```diff
@@ -0,0 +1,24 @@
+from foundry.smb3parse.util.code_edit import CodeEdit
+from foundry.smb3parse.util.rom import Rom
+
+class CodeEditByte(CodeEdit[int]):
+    """ Edit a single byte of memory in the ROM. """
+    def __init__(self, rom: Rom, start_address: int, prefix: bytearray, postfix: bytearray):
+        super().__init__(rom, start_address, 1, prefix, postfix)
+
+    def read(self) -> int:
+        """ Reads the target byte out of the ROM.
+        
+        This reads the byte at the target code address if both the prefix and
+        postfix are valid.  If they are not valid, returns None.
+        """
+        if not self.is_valid(): return None
```

Comment:
The read method promises that it will return an int; however, you return None in one of the cases.  Either read should promise to optionally return an int (`typing.Optional[int]`) or should raise a custom exception.